packages/shopping/src/services/authorizor.ts

```diff
@@ -0,0 +1,43 @@
+import {Provider, inject} from '@loopback/core';
+import {
+  Authorizer,
+  AuthorizationContext,
+  AuthorizationMetadata,
+  AuthorizationRequest,
+  AuthorizationDecision,
+} from '@loopback/authorization';
+import * as casbin from 'casbin';
+
+// Class level authorizer
+export class CasbinAuthorizationProvider implements Provider<Authorizer> {
+  constructor(@inject('casbin.enforcer') private enforcer: casbin.Enforcer) {}
+
+  /**
+   * @returns authenticateFn
+   */
+  value(): Authorizer {
+    return this.authorize.bind(this);
+  }
+
+  async authorize(
+    // @jannyHou: make sure it's universal for authorizers
```

Comment:
is it a reminder for something?  Either way, I think it's the best to remove the `@jannyHou`. :)

Reply:
Good catch...I think it's some notes from meeting, should be removed.

Reply:
agreed, removing note